taxi/forms.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import get_user_model
+from django import forms
```

Comment:
Add a blank line between different groups of imports to follow PEP-8 in the future